notes/health-care-evaluation.vbs

```diff
@@ -3997,360 +3952,606 @@ If HC_form_name = "No Form - Ex Parte Determination" Then
-			Text 10, 35, 100, 10, "- Income/Asset Renewal Date:"
-			Text 25, 45, 290, 20, "- For cases with a spenddown that do not meet an exception listed in EPM 2.3.4.2 MA-ABD Renewals, enter a date six months from the date updated in ELIG Renewal Date"
-			Text 25, 65, 275, 10, "- For all other cases, enter the same date entered in the Elig Renewal Date"
-			Text 10, 80, 145, 10, "- Exempt from 6 Mo IR: Enter N"
-			Text 10, 95, 145, 10, "- ExParte: Enter Y"
-			Text 10, 110, 255, 10, "- ExParte Renewal Month: Enter month and year of the ex parte renewal month"
-			EndDialog
+		'For ex parte denial, write information to case note
+		If ex_parte_determination = "Cannot be Processed as Ex Parte" Then
+			CALL write_variable_in_case_note("Phase 1 - The case has been evaluated for ex parte and has been denied based on the information provided.")
+			CALL write_bullet_and_variable_in_case_note("Reason for Denial:", ex_parte_denial_explanation)
+			'TO DO - add additional language listing what would qualify for ex parte?
+			'TO DO - add additional case details - case number, renewal info, etc?
+		End If
 
+		If ex_parte_determination = "Health Care has been Closed" Then
+			CALL write_variable_in_case_note("No renewal required on case as the Health Care is no longer active.")
+		End If
 
-			DO
-				Do
-					err_msg = ""    'This is the error message handling
-					Dialog Dialog1
-					cancel_confirmation
+		'Add worker signature
+		CALL write_variable_in_case_note("---")
+		CALL write_variable_in_case_note(worker_signature)
 
-					' If ButtonPressed = hc_renewal_button Then Call check_hc_renewal_updates() ' TO DO - timing of function calls and completing function within loop?
+		'Script end procedure
+		script_end_procedure("Success! The ex parte review information has been added to the CASE NOTE")
+	End If
 
-					'TO DO - update with functions?
-					'Check the HC renewal screen data and compare against initial to ensure that changes made properly
+	If ex_parte_phase = "Phase 2" Then
+		Dialog1 = ""
+
+		'TO DO - add functionality to determine ex parte phase based on case number
+		BeginDialog Dialog1, 0, 0, 216, 265, "Phase 2 - Ex Parte Denied"
+		GroupBox 10, 5, 200, 60, "Case Info"
+			Text 15, 20, 20, 10, "Case:"
+			Text 55, 20, 150, 10, case_number
+			Text 15, 35, 35, 10, "Person 1:"
+			Text 55, 35, 150, 10, name_01
+			If name_02 <> "" Then
+				Text 15, 50, 35, 10, "Person 2:"
+				Text 55, 50, 150, 10, name_02
+			End If
+		GroupBox 10, 70, 200, 65, "Ex Parte Denial Explanation"
+			Text 15, 85, 185, 20, "You are documenting an ex parte denial. Provide an explanation for the denial below."
+			EditBox 15, 110, 185, 15, ex_parte_denial_explanation
+		GroupBox 10, 140, 200, 75, "Update HC Renewal Date"
+			Text 15, 155, 190, 10, "Update the HC renewal date based on the ex parte denial."
+			Text 15, 175, 100, 10, "Updated HC Renewal Month:"
+			EditBox 110, 170, 20, 15, updated_hc_renewal_month
```

Comment:
This looks a little weird, the month an year are usually next to each other. They can be separate boxes but should be aligned side by side with a text field that indicates the month and year.

Reply:
Fixed - adjusted so that month and year are next to each other